app/models/user.rb

```diff
@@ -256,6 +257,11 @@ def default_country_to_usa
       self.country ||= "USA"
     end
 
+    def sanitize_postal_code
+      return unless self.postal_code
+      self.postal_code = self.postal_code.delete(' ').upcase
+    end
```

Comment:
The setter `self.postal_code=` is 👍 but there's no need here for `self`s where they are referencing the `postal_code` getters.